orangecontrib/text/widgets/owpubmed.py

```diff
@@ -0,0 +1,463 @@
+        self.pubmed_controls.append(self.advanced_query_input)
+
+        gui.createTabPage(self.search_tabs, 'Regular search',
+                          regular_search_box)
+        gui.createTabPage(self.search_tabs, 'Advanced search',
+                          advanced_search_box)
+
+        # Search info label.
+        self.search_info_label = gui.label(
+                self.controlArea, self,
+                'Number of retrievable records for this search query: /')
+
+        # Search for records button.
+        self.run_search_button = gui.button(
+                self.controlArea,
+                self,
+                'Find records',
+                callback=self.run_search,
+                tooltip='Performs a search for articles that fit the '
+                        'specified parameters.')
+        self.pubmed_controls.append(self.run_search_button)
+
+        h_line = QFrame()
+        h_line.setFrameShape(QFrame.HLine)
+        h_line.setFrameShadow(QFrame.Sunken)
+        self.controlArea.layout().addWidget(h_line)
+
+        # RECORD RETRIEVAL
+        # Text includes box.
+        text_includes_box = gui.widgetBox(self.controlArea,
+                                          'Text includes', addSpace=True)
+        self.authors_chbox = gui.checkBox(text_includes_box, self,
+                                          'includes_authors', 'Authors')
+        self.title_chbox = gui.checkBox(text_includes_box, self,
+                                        'includes_title', 'Article title')
+        self.mesh_chbox = gui.checkBox(text_includes_box, self,
+                                       'includes_mesh', 'Mesh headings')
+        self.abstract_chbox = gui.checkBox(text_includes_box, self,
+                                           'includes_abstract', 'Abstract')
+        self.pubmed_controls.append(self.authors_chbox)
+        self.pubmed_controls.append(self.title_chbox)
+        self.pubmed_controls.append(self.mesh_chbox)
+        self.pubmed_controls.append(self.abstract_chbox)
+
+        # Num. records.
+        h_box = gui.widgetBox(self.controlArea, orientation=0)
+        label = gui.label(h_box, self, 'Retrieve')
+        label.setMaximumSize(label.sizeHint())
+        self.num_records_input = gui.spin(h_box, self, 'num_records',
+                                          minv=1, maxv=100000)
+        self.max_records_label = gui.label(h_box, self, 'records from /.')
+        self.max_records_label.setMaximumSize(self.max_records_label
+                                              .sizeHint())
+        self.pubmed_controls.append(self.num_records_input)
+
+        # Download articles.
+        # Search for records button.
+        self.retrieve_records_button = gui.button(
+                self.controlArea,
+                self,
+                'Retrieve records',
+                callback=self.retrieve_records,
+                tooltip='Performs a search for articles that fit the '
```

Comment:
This tooltip is the same as for the `Find records` button. Since the buttons have different functionality their tooltips should differ.